docs/content/preview/yugabyte-voyager/ubuntu.md

```diff
@@ -19,7 +19,14 @@ Perform the following steps to install yb-voyager using apt for Ubuntu:
 
     This repository contains the `yb-voyager` Debian package and the dependencies required to run `yb-voyager`.
 
-1. Clean the `apt-get` cache and package lists using the following commands::
+1. Install the `postgresql-common` repository to fetch postgresql 16 using the following commands:
```

Comment:
In upgrade, this is only mentioned for RHEL, but in install it is only mentioned for Ubuntu?

Reply:
Sorry my commits got jumbled up. Fixing it.